### ISSUE TITLE
Add optional statistics

### ISSUE THREAD

**Issue by usrme** [usrme/comet-alt#11]
After browsing my Podcast Addict statistics it dawned on me that it might be neat to know how much time is spent writing commits. This would be wholly optional through the [configuration file](https://github.com/usrme/cometary/blob/master/comet.json) and I'm imagining allow for breakdowns per day, week, month, and year. The values would just be stored as integers (32 bit should be enough, giving a possible of 68 years of data) as running seconds.

I'll need to consider how to account for time spent in the `$EDITOR` if the ['specifyBody' option](https://github.com/usrme/cometary/blob/ec9111ca9cadd5eaac34a082029b480397ebd313/gui.go#L86) was chosen. My current assumption would be to just time the amount it takes for the [model to run](https://github.com/usrme/cometary/blob/ec9111ca9cadd5eaac34a082029b480397ebd313/main.go#L39-L53) and that should take into account all possible scenarios.

The statistics would be stored in a plain old text file (most likely in JSON) and showing the statistics could be done through a `--stats` flag, either prepended to the actual interface or stand-alone (this could be configurable).

I'm also imagining that some sort of toast could happen (e.g. confetti) if a defined number of seconds has passed for the day, week, month or year. This is a stretch goal.

**Comment by usrme** [comment]
I've made a few commits on the `issue-11` branch, but I'm once again getting bogged down in terms of performance, much like in #10. I'll have to have a look on how to improve things as I really don't want any noticeable delays when starting the program.

**Comment by usrme** [comment]
For future reference: the way to quit the program right after the first draw is to add the following:

```diff
--- a/gui.go
+++ b/gui.go
@@ -182,6 +182,7 @@ func (m *model) Init() tea.Cmd {
 }

 func (m *model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+       return m, tea.Quit
        switch msg := msg.(type) {
        case tea.KeyMsg:
                switch {
```

And the current benchmark off of `master` is:

```console
$ hyperfine --warmup 10 --output pipe --ignore-failure './cometary'
...
Time (mean ± σ):      47.8 ms ±   6.5 ms    [User: 11.1 ms, System: 13.5 ms]
  Range (min … max):    39.0 ms …  82.7 ms    52 runs
```

**Comment by usrme** [comment]
Looks like I'm on the home stretch with the latest commits and with no impact to the start-up time; the measurement is basically identical to the above 🎉

**Comment by usrme** [comment]
Resolved in [release v1.7.0](https://github.com/usrme/cometary/releases/tag/v1.7.0).
